tests/unit/django_spanner/models.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC
```

Comment:
FYI new files should be 2021 now

Reply:
updated to 2021 in all files added in 2021.